dm/templates/project/project.py

```diff
@@ -258,14 +258,14 @@ def create_service_accounts(context, project_id):
     network_list = [
         'serviceAccount:$(ref.{}-project.projectNumber)@cloudservices.gserviceaccount.com'.format(context.env['name'])
     ]
+    service_account_dep = ["{}-api-compute.googleapis.com".format(context.env['name'])]
```

Comment:
What if Compute API is not turned on? Shouldn't this be `service_account_dep = []`?

Reply:
Reworked